ui/src/app/components/render-resource-card.tsx

```diff
@@ -3,7 +3,7 @@ import * as fp from 'lodash/fp';
 
 import { BillingStatus, WorkspaceResource } from 'generated/fetch';
```

Comment:
Do you plan to remove this component in a future PR?

Reply:
Yes